packages/models/src/util.js

```diff
@@ -717,10 +717,14 @@ export function hashEvent(e) {
 
   const content = [];
   getStaticPropValues(e).forEach((v) => content.push(v));
-  e.parameters.forEach((p) =>
-    getStaticPropValues(p).forEach((v) => content.push(v))
-  );
-  [...e.labels].forEach((l) => content.push(l));
+  if (e.parameters) {
+    e.parameters.forEach((p) =>
+      getStaticPropValues(p).forEach((v) => content.push(v))
+    );
+  }
+  if (e.labels) {
+    [...e.labels].forEach((l) => content.push(l));
+  }
```

Comment:
Was this occurring? `labels` should always exist.

Reply:
The labels need not be copied into an array either, the `forEach` method will exist on the `Set`.